config/initializers/fog.rb

```diff
@@ -0,0 +1,15 @@
+CarrierWave.configure do |config|
+  if !Rails.env.test? && !Rails.env.development?
+    config.storage = :fog
+    config.fog_credentials = {
+    :provider               => 'AWS',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.